lib/web/apiserver.go

```diff
@@ -1401,11 +1401,18 @@ func (h *Handler) githubLoginWeb(w http.ResponseWriter, r *http.Request, p httpr
 		return client.LoginFailedRedirectURL
 	}
 
+	addr, err := utils.ParseAddr(r.RemoteAddr)
```

Comment:
Isn't `ParseAddr` intended for URIs? the `RemoteAddr` of a http request is guaranteed to be a `host:port`, isn't it?